contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -692,6 +692,62 @@ is_json_query(List *name)
 	}
 }
 
+/*
+* Parse T-SQL CONTAINS predicate. Currently only supports 
+* ... CONTAINS(column_name, '<contains_search_condition>') ...
+* This function transform it into a Postgres AST that stands for
+* to_tsvector('fts_contains', column_name) @@ to_tsquery('fts_contains', babelfish_fts_contains_rewrite('<contains_search_condition>'))
```

Comment:
references to 'fts_contains' in comment headers should be changed to reflect new behavior with the function that determines the correct config

Reply:
fixed